src/index.js

```diff
@@ -2,7 +2,11 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "semantic-ui-css/semantic.min.css";
 import "./index.css";
-import App from "./App";
 import "./i18n";
 
-ReactDOM.render(<App />, document.getElementById("root"));
+import configureFirebase from "services/Firebase";
+
+configureFirebase().then(async () => {
```

Comment:
If we could somehow do this inside `App` component then we won't need a hack for storybooks. Maybe it's possible with react's suspense, so we prevent app from actually rendering until we have firebase connection? 🤔

Reply:
yep, we could use `Suspense` with some kind of loader. Have you checked how long it usually takes to configure?

Reply:
It seems to be pretty much instant (it's only downloading a single small json file from google servers). But it needs to block app rendering (or even more likely it's just loading itself, not even rendering - Auth and DataStore try to configure firebuse upon import), otherwise we try to use unconfigured firebase object which raises an error.

Reply:
And I may have tried using Suspense in a wrong way, but it seemed to me like it's always initializing all components even when they aren't rendered yet, which resulted in firebase error anyway

Reply:
That looks exactly like a kind of solution I was trying to implement. I'll take a look at it later

Reply:
I like this library, but it seems like using it will require quite a big rewrite of how we manage data fetching and authorization. I'd rather do this in a separate issue